test/extended/imagepolicy/imagepolicy.go

```diff
@@ -142,6 +148,106 @@ var _ = g.Describe("[sig-imagepolicy][OCPFeatureGate:SigstoreImageVerification][
 	})
 })
 
+var _ = g.Describe("[sig-imagepolicy][OCPFeatureGate:SigstoreImageVerificationPKI][Serial]", g.Ordered, func() {
```

Comment:
Per new OTE naming conventions we should add "[Jira:Node]"  for clear ownership.  This is how Compnent Readiness knows what component to assign tests to.

Reply:
I agree, I can do a follow-up to clarify all the annotations under this file.